packages/css/src/alert/alert.scss

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-alert {
+  align-items: flex-start;
+  background-color: var(--amsterdam-alert-background-color);
+  border-color: var(--amsterdam-alert-border-color);
+  border-style: var(--amsterdam-alert-border-style);
+  border-width: var(--amsterdam-alert-border-width);
+  display: flex;
+  flex-direction: row;
+  gap: var(--amsterdam-alert-gap);
+  justify-content: space-between;
+  padding-block-end: var(--amsterdam-alert-padding-block-end);
+  padding-block-start: var(--amsterdam-alert-padding-block-start);
+  padding-inline-end: var(--amsterdam-alert-padding-inline-end);
+  padding-inline-start: var(--amsterdam-alert-padding-inline-start);
+}
+
+.amsterdam-alert__content {
+  flex-basis: 100%;
+}
+
+.amsterdam-alert__title {
+  font-family: var(--amsterdam-alert-title-font-family);
+  font-size: var(--amsterdam-alert-title-narrow-font-size);
+  font-weight: var(--amsterdam-alert-title-font-weight);
+  line-height: var(--amsterdam-alert-title-line-height);
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-alert-title-wide-font-size);
+  }
+}
+
+.amsterdam-alert__text {
+  display: flex;
+  flex-direction: column;
+  font-family: var(--amsterdam-alert-text-font-family);
+  font-size: var(--amsterdam-alert-text-narrow-font-size);
+  font-weight: var(--amsterdam-alert-text-font-weight);
+  line-height: var(--amsterdam-alert-text-line-height);
+  row-gap: var(--amsterdam-alert-text-row-gap);
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-alert-text-wide-font-size);
+  }
+}
```

Comment:
All of this should be removed. If `children` are used, they are responsible for their layout and appearance.